static/js/typeahead_helper.js

```diff
@@ -390,9 +389,7 @@ exports.compare_by_activity = function (stream_a, stream_b) {
     if (diff !== 0) {
         return diff;
     }
-    diff =
-        peer_data.get_subscriber_count(stream_b.stream_id) -
-        peer_data.get_subscriber_count(stream_a.stream_id);
+    diff = (stream_b.stream_weekly_traffic || 0) - (stream_a.stream_weekly_traffic || 0);
```

Comment:
This function has a comment on top that's now wrong.  Otherwise this commit lgtm.

Reply:
I'll fix the comment in the next push.